docs/ProductConstructionServiceDevGuide.md

```diff
@@ -0,0 +1,3 @@
+# Starting the service locally
```

Comment:
Can you merge this with `instructions.md` and call it `Readme.md` and just put it in the PCS folder?

Reply:
done